internal/config_checks_generic.h

```diff
@@ -126,7 +126,7 @@ MOZZI_CHECK_POW2(MOZZI_CONTROL_RATE)
 #error "MOZZI_AUDIO_INPUT depends on MOZZI_ANALOG_READ option"
 #endif
 
-#if !MOZZI_IS(MOZZI_AUDIO_INPUT, MOZZI_AUDIO_INPUT_NONE) && defined(MOZZI_AUDIO_INPUT_PIN)
+#if MOZZI_IS(MOZZI_AUDIO_INPUT, MOZZI_AUDIO_INPUT_NONE) && defined(MOZZI_AUDIO_INPUT_PIN)
```

Comment:
This one was correct before: Shows a warning, if MOZZI_AUDIO_INPUT_PIN is defined without MOZZI_AUDIO_INPUT

Reply:
Are you sure?
 
The old one is: If Mozzi_input is Not None (hence, it is defined) and a pin is defined, shows an error. This gives a warning when compiling the Audio_Input example.

The new one is: If Mozzi_input is None and a pin is defined, shows an error.

Reply:
You are right!